dev/com.ibm.ws.concurrent/src/com/ibm/ws/concurrent/internal/ManagedThreadFactoryService.java

```diff
@@ -151,7 +152,7 @@ public class ManagedThreadFactoryService implements ResourceFactory, Application
     private ThreadGroupTracker threadGroupTracker;
 
     /**
-     * Virtual thread operations that are only available when a Java 21+ feature includes the io.openliberty.threading.internal.java21 bundle.
+     * Virtual thread operations that are only available when a Java 21+ feature includes the io.openliberty.threading.virtual.internal bundle.
      */
     @Reference(cardinality = ReferenceCardinality.OPTIONAL,
```

Comment:
This reference can become static/mandatory now I think.  Also the comment needs updated as we always will have a `VirtualThreadOps` registered regardless of the java version.